lib/logger/notes.md

```diff
@@ -0,0 +1,42 @@
+## Distributed Logger Proposal
+
+### Issue Addressed
+
+As Elixir apps are deployed across dozens of nodes, it becomes harder to
+manage and inspect all the independent log files. Some, like Bleacher
+Report, resort to [third party solutions](https://dev.bleacherreport.com/elixir-phoenix-centralized-http-logging-aa50efe3105b).
+
+This seems unnecessary, as this kind of distributed solution is square
+in Elixir's sweet spot.
+
+### Proposal
+
+1. Alter the `console` logger backend to accept a new configuration
+   parameter, `known_as`. This would publish the a name for this logger.
+   The name could be an atom, is which case `global` is used, or it
+   could be a `:via` tuple.
```

Comment:
There is a mismatch here. The logger backends are not named, only the Logger `gen_event`. So we would have to name the whole logger backend.

My suggestion is to call this option `:name` or something that ends with `_name` and keep the same naming arguments as a GenServer for consistency. So `name: Logger` would be the default, `name: {:global, ...}` or via would be supported too.